### ISSUE TITLE
Check the document for editorial errors

### ISSUE THREAD

**Issue by cpandhi** [w3c/wcag-act#9]
https://www.w3.org/2002/09/wbs/35422/ACT_DPUB_Review_Nov_2016/results

**Comment by moekraft** [comment]
Editorial work needed to convert Markdown format, hard to follow links in its current form. - Michael Cooper.

**Comment by moekraft** [comment]
[Accessibility Conformance Testing (ACT) draft.docx](https://github.com/w3c/wcag-act/files/631453/Accessibility.Conformance.Testing.ACT.draft.docx)

-Feedback from Mike Gower
